test/libsolidity/SyntaxTest.cpp

```diff
@@ -166,12 +166,12 @@ void SyntaxTest::printErrorList(
 )
 {
 	if (_errorList.empty())
-		FormattedScope(_stream, _formatted, {BOLD, GREEN}) << _linePrefix << "Success" << endl;
+		AnsiColorized(_stream, _formatted, {BOLD, GREEN}) << _linePrefix << "Success" << endl;
 	else
 		for (auto const& error: _errorList)
 		{
 			{
-				FormattedScope scope(_stream, _formatted, {BOLD, (error.type == "Warning") ? YELLOW : RED});
+				AnsiColorized scope(_stream, _formatted, {BOLD, (error.type == "Warning") ? YELLOW : RED});
```

Comment:
I wonder why here it was used as local variable and not just ``<<``-ed into as everywhere else :-). But unrelated to this PR.